### PR TITLE
OLD) [LC-600] Add CI for MainNet Sync

### PR DESCRIPTION
**Pull request by metalg0su**
# Goal
- Add CI to check citizen -> MainNet sync

-----
> Sorry. I mistaked opening this PR with wrong number.
> Please see: https://github.com/icon-project/loopchain/pull/488

### CODE REVIEW

**Comment by yakkle** [comment]
Is this test for mainnet sync while 20 seconds?

File: .github/workflows/citizen.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    - name: Generate-key
+      run: |
+        sed "23 a\  \"PRIVATE_PASSWORD\": \"${pwd}\"," -i conf/mainnet/loopchain_conf.json
+        cat conf/mainnet/loopchain_conf.json
+
+        pip install tbears
+        tbears keystore -p ${pwd} my_keystore.json
+      env:
+        pwd: Most_safest_pw_in_the_wor1d
```
Comment (yakkle) [inline .github/workflows/citizen.yml]: most safest! 😄

Comment (metalg0su) [reply .github/workflows/citizen.yml]: > Is this test for mainnet sync while 20 seconds?

The test scenario is summarized:
- Run Loopchain to sync
- Give a short time to the node to **reach the target block height.**
- Ask to node whether it reached to the target or not.

The waiting time is essential to reach the target block height.
But there's no reason for the time to be `20` sec.

Comment (metalg0su) [reply .github/workflows/citizen.yml]: Sorry. I mistaked opening this PR with wrong number.

Please see: https://github.com/icon-project/loopchain/pull/488

**Comment by metalg0su** [comment]
Sorry. I mistaked opening this PR with wrong number.

Please see: https://github.com/icon-project/loopchain/pull/488

**Pull request closed without merging**